### DISCUSSION TITLE
Keyboard Shortcuts and Screen Demonstration

### DISCUSSION THREAD

**Discussion by FxnyrShiro** [stoatchat/discussions#821] [Feature Suggestions]
Keyboard shortcuts to mute the microphone, sound, and reverse walkie-talkie (Push to talk on the contrary)

And so that the screen can be demonstrated, and the discord is no longer needed

**Comment by EcoGive** [comment]
I agree that the walkie-talkie mode should be added.

**Comment by tgoncuoglu** [comment]
I would love to see "toggle mute" and "push to talk" keybinds in the app.
